EIPS/eip-TBD.md

```diff
@@ -1,85 +1,57 @@
-
-  With a few exceptions, external links are not allowed. If you feel that a particular resource would demonstrate a compelling case for your EIP, then save it as a printer-friendly PDF, put it in the assets folder, and link to that copy.
-
-  TODO: Remove this comment before submitting
--->
+We describe a game-theoretic mechanism that targets the primary actors in a transfer (i.e. the previous and new token owner) as the only players, and induce them to truthfully supply the information required for enforcement.
 
 ## Specification
 
-<!--
-  The Specification section should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (besu, erigon, ethereumjs, go-ethereum, nethermind, or others).
-
-  It is recommended to follow RFC 2119 and RFC 8170. Do not remove the key word definitions if RFC 2119 and RFC 8170 are followed.
-
-  TODO: Remove this comment before submitting
--->
-
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
 
-For each `tokenId` we introduce a set of addresses whose members MAY transfer the token to themselves without payment by calling `TBD()`, which we will refer to as *take-back*.
+For each `tokenId` we introduce a set of addresses whose members MAY transfer the token to themselves without payment by calling `TBD()`, which we will refer to as a *take-back*.
 The set MUST be initially empty.
-For every [ERC-721](./eip-721.md) transfer, *except* for initial mint, the set of the corresponding `tokenId` MUST be appended with the previous owner (i.e. the transfer's `from` address).
+For every ERC-721 transfer, *except* for initial mint, the set of the corresponding `tokenId` MUST be appended with the previous owner (i.e. the transfer's `from` address).
 
-Royalty payment by the token owner MUST be accepted by the token contract via a dedicated function, `TBD()`.
+Royalty payment by the new token owner MUST be accepted by the token contract via a dedicated function, `TBD()`.
 Upon receipt of royalties, the take-back set MUST be emptied (i.e. transfer permissions collapse back to standard ERC-721 rules).
-After each token transfer, there SHOULD be a grace period during which the token is frozen and take-backs are disabled to allow the new owner to decide whether to pay the royalty or allow the take-back set to grow.
+After each token transfer, there SHOULD be a grace period during which the token is frozen and take-backs are disabled to allow the new owner to decide whether to either pay the royalty or allow the take-back set to grow.
```

Comment:
Do you still need to use "either" if you already have "whether"?

Reply:
Not strictly necessary but it aids the reader in knowing that there's a binary split coming up, instead of them realising this halfway through the sentence. It's like using a `switch` instead of a series of `if else`; same end goal but better signalling of intent.

Reply:
I think I've just never seen `whether to either`. To me both signal "one of the following".